src/Npc.ts

```diff
@@ -6,25 +6,26 @@ import { IGameState } from './GameState';
 import { IItemDef } from './Item';
 import { Logger } from './Logger';
 import { Room } from './Room';
-import { Scriptable } from './Scriptable';
 
 const uuid = require('uuid');
 
 export interface INpcDef extends ICharacterConfig {
 	area?: string;
 	script?: string;
 	behaviors?: Record<string, any>;
-	equipment?: Record<string, IItemDef> | Record<string, { entityReference: string; }>;
+	equipment?:
+		| Record<string, IItemDef>
+		| Record<string, { entityReference: string }>;
 	items?: EntityReference[];
 	description: string;
 	entityReference: EntityReference;
-	id: string | number;
+	id: string;
 	keywords: string[];
 	quests?: EntityReference[];
 	uuid?: string;
 }
 
-export class Npc extends Scriptable(Character) {
```

Comment:
Scriptable is already in Character and this was causing some circular dependency issues I believe

Reply:
I think you're right about this -- I forget if I fixed this the same way locally but I definitely ran into it. Good call